src/components/ResultsListItem.vue

```diff
@@ -91,5 +96,9 @@ export default {
   .disabled {
     pointer-events: none;
   }
+  
+  @media screen and (max-width: 360px) {
```

Comment:
These media queries are making things look much better on desktop. Thanks for all your work on this